<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup</class>
 <widget class="QDialog" name="Signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>400</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Sign up</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(24, 24, 24);</string>
  </property>
  <widget class="QWidget" name="widget_Main" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>351</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>344</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_SignUp">
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="title">
       <property name="styleSheet">
        <string notr="true">font: 63 18pt &quot;Segoe UI Semibold&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Sign up</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="grid_SignUp">
       <item row="2" column="0">
        <widget class="QLabel" name="label_Password">
         <property name="styleSheet">
          <string notr="true">font: 63 14pt &quot;Segoe UI Semibold&quot;;
color: rgb(226, 148, 24);</string>
         </property>
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="button_SignUp">
         <property name="styleSheet">
          <string notr="true">font: 63 10pt &quot;Segoe UI Semibold&quot;;
background-color: rgb(52, 52, 52);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Sign up</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_Username">
         <property name="styleSheet">
          <string notr="true">font: 63 14pt &quot;Segoe UI Semibold&quot;;
color: rgb(226, 148, 24);</string>
         </property>
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_Username">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_Password">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="layout_SignIn">
         <item>
          <widget class="QLabel" name="label_Question">
           <property name="styleSheet">
            <string notr="true">font: 63 10pt &quot;Segoe UI Semibold&quot;;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Already have an account ?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_SignIn">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">text-decoration: underline;
font: 63 10pt &quot;Segoe UI Semibold&quot;;
color: rgb(226, 148, 24);</string>
           </property>
           <property name="text">
            <string>Sign in</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
